kwalitee/kwalitee.py

```diff
@@ -141,17 +142,26 @@ def _check_bullets(lines, **kwargs):
     """
     max_length = kwargs.get("max_length", 72)
     labels = {l for l, _ in kwargs.get("commit_msg_labels", tuple())}
+    signatures = kwargs.get("signatures", tuple())
+
+    def _is_signature(line):
```

Comment:
To properly find the end of the bullet message, as the signatures don't have a dot they should be treated differently and never be considered part of the dotted message itself.